server/src/routes/api/v1/podcastReviewsRouter.js

```diff
@@ -12,17 +13,19 @@ podcastReviewsRouter.post('/', async (req, res) => {
   const formInput = cleanUserInput(body)
   const { description, rating } = formInput
   const { podcastId } = req.params
+  const userId = req.user.id
 
   try {
     const newReview = await Review.query().insertAndFetch({
-      description, rating, podcastId
+      description, rating, podcastId, userId
     })
+    newReview.user = UserSerializer.getSummary(req.user)
```

Comment:
This new Review should be passed through a serializer where we can then set this association of `newReview.user`